rs/xnet/payload_builder/src/lib.rs

```diff
@@ -1227,7 +1300,7 @@ pub const POOL_SLICE_BYTE_SIZE_MAX: usize = 4 << 20;
 
 /// Conservative minimum slice size in bytes. We stop trying to add slices to
 /// the payload once we're this close to the payload size limit.
-pub const SLICE_BYTE_SIZE_MIN: usize = 1 << 10;
+pub const SLICE_BYTE_SIZE_MIN: usize = 600;
```

Comment:
Is this really big enough -- especially considering that the hash tree and the certification are also part of the signature?

Reply:
Is this size important for correctness or is it just used for optimization purposes?